packages/venia-ui/lib/components/Modal/index.js

```diff
@@ -1 +1,3 @@
-export { default as Modal } from './modal';
+// For now, Modal is an alias for Portal.
+// This is deprecated behavior and may be changed in the future.
+export { Portal as Modal } from '../Portal';
```

Comment:
I don't think Modal was part of the last release so I don't think this would be needed.

Reply:
Unfortunately `Modal` has been around for a while (at least `5.0.0`), so we're not sure who may be using it. I'll be sure to bring this up in community sync.